tests/test_text_generation_example.py

```diff
@@ -117,7 +128,16 @@ def _test_text_generation(
         pattern = re.compile(r"([\"\'].+?[\"\'])|\s")
         command = [x for y in command for x in re.split(pattern, y) if x]
 
-        proc = subprocess.run(command, env=env_variables)
+        if fp8:
+            os.environ["QUANT_CONFIG"] = os.path.join(
+                path_to_example_dir, "text-generation/quantization_config/maxabs_measure_include_outputs.json"
```

Comment:
we can't have env set this way as it breaks all other test. why didn't use env_variable?

Reply:
I agree, we should update `env_variables` here